ipaserver/install/cainstance.py

```diff
@@ -1727,6 +1728,33 @@ def ensure_lightweight_cas_container():
     )
 
 
+def minimum_acme_support(data=None):
+    """
+    ACME with global enable/disable is required.
+
+    This first shipped in dogtag version 10.10.0.
+
+    Parse the version string to determine if the minimum version
+    is met. If parsing fails return False.
+
+    :param: data: The string value to parse for version. Defaults to
+                  reading from the filesystem.
+    """
+    if not data:
+        with open('/usr/share/pki/VERSION', 'r') as fd:
+            data = fd.read()
+
+    groups = re.match('.*\nSpecification-Version: ([\d+\.]*)\n.*', s)
+    if groups:
+        version_string = groups.groups(0)[0]
+        minimum_version = parse_version('10.10.0')
+
+        return parse_version(version_string) >= minimum_version
+    else:
+        logger.debug('Unable to parse version from {}'.format(data))
+        return False
+
```

Comment:
@frasertweedale change in plans. I think we may ship with this minimum version test so we can merge this without breaking until pki 10.10.0 lands. I changed the file test into a version parser which should be less fragile.